### ISSUE TITLE
CLI

### ISSUE THREAD

**Issue by RobLoach** [jonschlinkert/falsey#2]
Easy way to find if something is falsey through through command line... Exits with error when not `falsey`.

**Comment by jonschlinkert** [comment]
good idea! is your preference for it to be built-in, or externalized to something like `falsey-cli`?

**Comment by RobLoach** [comment]
One `cli.js` with no dependencies? Don't see why it would need the separation, but I'll leave that for you to decide.

**Comment by jonschlinkert** [comment]
nope that answered my question lol. If it had a big dependency tree like with meow or something, I'd externalize it.  otherwise I generally like the idea of having it included
